index.html

```diff
@@ -0,0 +1,521 @@
+                convallis iaculis sapien at pretium.
+              </p>
+            </div>
+          </div>
+        </a>
+      </section>
+      <!-- Jobs section -->
+      <section class="max-w-screen-xl mx-auto px-4">
+        <div class="max-w-xl py-12">
+          <h2 class="text-black text-3xl font-bold">
+            We’re always looking for awesome people to join us
+          </h2>
+          <p class="mt-6 text-base text-black">
+            Diam nunc lacus lacus aliquam turpis enim. Eget hac velit est
+            euismod lacus. Est non placerat nam arcu. Cras purus nibh cursus sit
+            eu in id.
+          </p>
+        </div>
+        <div class="grid grid-cols-2 gap-x-8">
+          <div class="flex justify-center items-center">
+            <img
+              src="/assets/images/image-1200x1200 (1).jpg"
+              alt="Bag with accessories"
+              width="600"
+              height="600"
+            />
+          </div>
+          <div>
+            <div class="p-4 space-y-4 border-b border-b-gray-200">
+              <span>Full-time designer</span>
+              <p class="text-sm">
+                Quos sunt ad dolore ullam qui. Enim et quisquam dicta molestias.
+                Corrupti quo voluptatum eligendi autem labore.
+              </p>
+              <div class="flex space-x-8">
+                <span class="block">£80,000</span>
+                <span class="block">Manchester, UK</span>
+              </div>
+            </div>
+            <div class="p-4 space-y-4 border-b border-b-gray-200">
+              <span>Full-time designer</span>
+              <p class="text-sm">
+                Quos sunt ad dolore ullam qui. Enim et quisquam dicta molestias.
+                Corrupti quo voluptatum eligendi autem labore.
+              </p>
+              <div class="flex space-x-8">
+                <span class="block">£80,000</span>
+                <span class="block">Manchester, UK</span>
+              </div>
+            </div>
+            <div class="p-4 space-y-4 border-b border-b-gray-200">
+              <span>Full-time designer</span>
+              <p class="text-sm">
+                Quos sunt ad dolore ullam qui. Enim et quisquam dicta molestias.
+                Corrupti quo voluptatum eligendi autem labore.
+              </p>
+              <div class="flex space-x-8">
+                <span class="block">£80,000</span>
+                <span class="block">Manchester, UK</span>
+              </div>
+            </div>
+            <div class="p-4 space-y-4">
+              <span>Full-time designer</span>
+              <p class="text-sm">
+                Quos sunt ad dolore ullam qui. Enim et quisquam dicta molestias.
+                Corrupti quo voluptatum eligendi autem labore.
+              </p>
+              <div class="flex space-x-8">
+                <span>£80,000</span>
+                <span>Manchester, UK</span>
+              </div>
```

Comment:
Unsure what we agreed on - should individual blog posts be clickable here?

Reply:
amended these to suit more blog posts